samples/Mvc.FileUpload/MultipartRequestHelper.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.IO;
+using Microsoft.Net.Http.Headers;
+
+namespace Mvc.FileUpload
+{
+    public static class MultipartRequestHelper
```

Comment:
@Tratcher - this seems like this is really a lot of code to copy into a sample. Do we have a more recommended approash?